lookup/api/views.py

```diff
@@ -15,10 +15,10 @@
 
 class LookupViewSet(viewsets.ModelViewSet):
     """Provide CRUD + L functionality for Lookup."""
-    authentication_classes = [TokenAuthentication] # Authenticate access to API by issueing a token
+    # authentication_classes = [TokenAuthentication] # Authenticate access to API by issueing a token
     queryset = Visitor.objects.all().order_by("-created_at")
     serializer_class = VisitorSerializer
-    permission_classes = [IsAuthenticated] # Allows access only to authenticated users.
+    permission_classes = [IsAdminUser] # Allows access only to authenticated users.
```

Comment:
Commeting out TokenAuthentication and using IsAdminUsr as a permission class, admin users other than a superuser lose eligibility to delete visitor data